operator/v1/types_ingress.go

```diff
@@ -665,6 +675,25 @@ type IBMLoadBalancerParameters struct {
 	Protocol IngressControllerProtocol `json:"protocol,omitempty"`
 }
 
+// OpenStackLoadBalancerParameters provides configuration settings that are
+// specific to OpenStack load balancers.
+type OpenStackLoadBalancerParameters struct {
+	// loadBalancerIP specifies the floating IP address that the load balancer will use.
+	// A user can use that so they can specific which address to use for a given load balancer
+	// instead of getting a random one from the cloud-provider-openstack.
```

Comment:
I'm not reading this sentence well, but, perhaps a more direct approach would be
```
When not specified, an IP address will be assigned randomly by the OpenStack cloud provider.
```